active/0000-nested-addon-imports.md

```diff
@@ -0,0 +1,63 @@
+- Start Date: (fill me in with today's date, 2016-08-06)
+- RFC PR: 
+- ember-cli Issue: 
+
+# Summary
+
+The new asset import API, available since ember-cli 2.7.0, offers the possibility to 
+import Ember CLI addons from Ember CLI addons (aka, nested addons).
+This RFC intends to find solutions for typical edge cases.
+
+# Motivation
+
+Since ember-cli 2.7.0, addons can be nested if the nesting floows the following
+scenario:
+* The nested addons do not make use of the property `parent.bowerProperty` to define
+  assets paths because it not always defined (it is not defined in EmberAddon instances)
+* The nested addons do not need to execute a default blueprint.
+
+Solving these use cases would allow for a simpler, more generic, addons development
+process, and will allow teams to build more reusable and higher level addons for
+their projects and the community (like [ember-admin](https://github.com/DockYard/ember-admin)
+using [ember-power-select](ember-power-select.com) with no additional boilerplate, for instance).
+
+# Detailed design
+
+Here is a rundown of the questions raised with @nathanhammond in [a previous PR](https://github.com/ember-cli/ember-cli/pull/6111):
+
+1. Should nested addons be allowed to use their grandparents' `bower_components` directory?
+1. How/When should the default blueprints of the nested be run?
+1. How to deal with conflicting versions (The app uses my-addon 1.0.0, but my-addon 2.0.0 is required as a nested component)?
+
+For each of these questions, here's a first attempt towards an answer:
+
+1. Yes, they should, because there's only one bower directory available to the app, and it should be available to the whole app,
+  including to the dependencies. See [the PR linked above](https://github.com/ember-cli/ember-cli/pull/6111) for an example.
```

Comment:
Having spent more time thinking about this, I'm opposed to this for reasons of version conflicts. Addons could attempt to `bower install` something which is in direct conflict with the host applications' dependencies. Further we'd be encouraging further adoption of `bower` when our intent is to move farther away from that.

In general if we do this I believe that the addon itself should receive its own separate `bower_components` directory.

This in general creates the situation where the host needs to be aware of the addon's dependencies in order to not conflict, which I think breaks the degree of isolation that we should be aiming for.